recipes-core/images/seapath-guest-common.inc

```diff
@@ -9,9 +9,11 @@ inherit ${@bb.utils.contains('DISTRO_FEATURES', 'seapath-security', 'security/qa
 
 IMAGE_INSTALL_append = " \
     cukinia-tests-vm \
-    pacemaker-remote \
```

Comment:
Why we removed packame-remote ?

Reply:
There is currently to use case for this package.